stan/math/opencl/kernel_generator/operation_cl_lhs.hpp

```diff
@@ -123,6 +124,20 @@ class operation_cl_lhs : public operation_cl<Derived, Scalar, Args...> {
           (this->template get_arg<Is>().add_write_event(e), 0)...};
     });
   }
+
+  /**
+   * Adds all read and write events on any matrices used by nested expressions
+   * to a list and clears them from those matrices.
+   * @param[out] events List of all events.
+   */
+  inline void get_clear_read_write_events(
+      std::vector<cl::Event>& events) const {
+    index_apply<N>([&](auto... Is) {
+      (void)std::initializer_list<int>{
```

Comment:
Should this be `static_cast<void>()`? I'm surprised this is getting through the cpplint

Reply:
Yeah. Fixed.